### DISCUSSION TITLE
None of extension disablers work (i think) - any solutions?

### DISCUSSION THREAD

**Discussion by wea-f** [3kh0/ext-remover#723] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
My school has an extension called xFanatical Safe Doc that gives us a 5 tab limit (So new ltmeat doesn't work) and the thing is that I want to disable it.
Also my version is 115, so LTBEEF doesn't work

ok my school blocked bookmarklets, when you try it nothing happens and you can't execute javascript, it'll just delete the text in the url box.
chrome://hang and chrome://kill are blocked - when you do it, it'll just delete the text in the url box and nothing happens
dextensify doesn't cover my blocker extension (CK Authenticator G3) nor any extention that i have and I can't execute javascript 
My school disabled inspect

when i go to https://chrome.google.com/webstorex, it says error 404 not found, also chrome100.dev is blocked
chrome://settings/signOut doesn't work - it just redirects you to chrome://settings
If I powerwash my chromebook (esc+refresh+power)  I can't access my school's wifi and can't access the education links and stuff so i can't do that
the incognito thingie doesn't work - no notification after pasting the IP server in custom name servers
chrome://chrome-signin doesn't work, I put the gmail in however the next page says: "This account is not allowed to sign in within this network.
Please talk to your network administrator for more information."
and the School Network Restriction Bypass IP servers don't work for me.
and sh1mmer.me is blocked
**THERE IS NO "ACCESS TO FILE URLS" SWITCH FOR THE EXTENSION**

**tldr: Any way to disable or temporarily disable an extension with less than 5 tabs?**

**Comment by Su4o** [comment]
Try This 
https://github.com/HENRYMARTIN5/ChromeBox
Yes tell the admin to download the app. they will :)

**Reply by wea-f** [reply]
My school blocks me from signing in with my gmail and don't think my admin will allow me to download the app.

**Reply by Su4o** [reply]
Ok then try one of these. It should work (even if you cant sign in with your gmail.)

Unblocked Google

Step 1. Go to chrome://chrome-signin 
Step 2. Click OK on the bottom right corner 
Step 3. In the Email text box put google@d11.org 
Step 4. Click signin options 
Step 5. Now click sign in with GitHub Step 
6. Click the GitHub cat icon Step 
8. In the search box on the top right type google and then click see more topics then you will see all the Google links click www.google.com now boom unblocked Google.

-------------------------

Baby NEW LTMEAT Disable extensions

First of all, get your folder with 800+ extension page tabs and open it in a new window, for my Chromebook i used 800 extension page tabs as i feel it's the right amount for me

Close the newly opened Window with 800+ extension page tabs

Click into your folder, and open one of the extension page tab in a new window, maybe waiting slightly longer this time, to confirm it worked. If it loads, you did it wrong. If you see a "page unresponsive screen, and a wait/exit page button" you did it right. BUT DO NOT CLICK THE EXIT PAGE BUTTON OR WAIT BUTTON. (Or if you want to do it fast you can just see that the page always has a spinning loading circle)

Now go to chrome://extensions/?id=mmhppocgfdombmapajfeafhecobfcjpk Then scroll down and flick the "allow access to file URLs" lever and close the window with the 1 extension page tab remaining.
-------------------------

NEW LTMEAT Disable extensions

If your Chromebook has received the 115 And Above patch on the stable channel Then Here's A New Method For LTMEAT
Create a bookmark folder and paste the extension page lots of times. (About 800 minimum is recommended assuming your Chromebook is average school quality) It is recommended that you add the extension page at the beginning of the folder.

Right click and open all in a new window.

Close the window with all those tabs.

Open the folder in a new window again, and Chrome should hang those tabs to take care of the old ones in the background that were just closed. (Equivalent to the duplicate tab step in Bypassi's method)

Flip the Allow access to file URLs switch in the extension settings and then you've bypassed the patch and the exploit is working.

Close everything and you're good to go. If it didn't work, try adjusting the number of tabs being opened. Bypassi should add this to his website as Alternate Method # 2. With this workaround, LTMEAT can endure a longer life!
-------------------------
Blank3r Reload extensions continuously
Point Blank is an exploit that allows you to run bookmarklets on privileged pages, such as the Chrome extensions page. This exploit was made with Point Blank as well.

The exploit code is below.

Bookmark this code:
javascript:let shim = false;var ids = prompt("extension ids (comma separated)").split(",");setInterval(()=>{ids.forEach((id)=> opener.chrome.developerPrivate.updateExtensionConfiguration({extensionId: id, fileAccess: shim}));shim = !shim;}, 145);
And the GUI is in launcher.js

Navigate to chrome://extensions.

Click on an extension that YOU installed from the Chrome Web Store > Details. **(If you don't have a extension that you have, just use any that lets you "view in chrome" **

In the URL bar, copy the string of letters and numbers after the /?id=.

Click "View in Chrome Web Store" and **spam the escape key**. If it loads into Chrome Webstore try again, if it is a blank screen click the bookmarklet.

Paste the id of the extension into the prompt separated by commas.

If you close the tab, the exploit will stop working.

**profit**

**Reply by wea-f** [reply]
None of these options work.
When I put the email, I don't see a sign-in option, I only see a "Next" button. If I press the "Next" button, it will lead me to the "Couldn't Sign you in " page where it says, "This account is not allowed to sign in within this network.
Please talk to your network administrator for more information."
![Screenshot 2023-10-10 10 02 44 AM](https://github.com/3kh0/ext-remover/assets/100387337/3e475477-70d6-4f2a-9b3f-af8eb6b45e5f)
LTMEAT doesn't work because I can't have more than 5 tabs (when I do it, my chromebook freezes and when it unfreezes, all of the opened tabs are gone and nothing happened to the extension page. ) plus Safe Doc doesn't have the "allow access to file URLs" option. In the "Site Access" tab it says "This extension has no additional site access." 
Blank3r doesn't work because my school disabled bookmarklets

**Reply by HENRYMARTIN5** [reply]
PSA: no, you should NOT use ChromeBox - it's unfinished and messy and really quite terrible, there are many other, objectively better ways that are functionally identical

**Comment by AshtonDavies** [comment]
Safe doc is to filter Google docs searches. I have it too, and it doesn't get in the way of doing this stuff.

**Reply by wea-f** [reply]
For my chromebook, it does. Before chrome://hang and chrome://kill got blocked, I did the LTMEAT to Safe Doc and I could open more than 5 tabs. Plus Safe Doc can limit browser tabs set by the admin. (https://xfanatical.com/blog/prevent-students-from-opening-too-many-chrome-tabs/#:~:text=Limit%20the%20number%20of%20Chrome,by%20old%20tabs%20are%20recycled)

**Comment by S-PScripts** [comment]
yes, i was having the same issue with the unblocked google method

**Comment by S-PScripts** [comment]
also ltmeat crashes my chromebook :p

**Reply by wea-f** [reply]
you should probably decrease the number of extension pages in the folder then

**Reply by S-PScripts** [reply]
oh yeah btw i did put less tabs but then no pages would load after doing ltmeat

**Reply by AshtonDavies** [reply]
> oh yeah btw i did put less tabs but then no pages would load after doing ltmeat

That means you didn't get the obscure looking hanged tabs from reopening the folder.

**Comment by Su4o** [comment]
ok hm.
if your school isn't that strict (but they already probably blocked this) 

use **hyperbeam**. only problem i see here is that you have to use a temp email because you can't sign in with google. or try to sign in with a different device for hyperbeam.

**Reply by S-PScripts** [reply]
[REDACTED]

The site was blocked by restriction list: [REDACTED](http://www.rm.com/filteringpolicy/)
Contact [EMAIL](mailto:EMAIL) for queries regarding filtering.

https://hyperbeam.com/

RM Education Copyright © RM 2023 [Terms](http://www.ifl.net/safetynet/terms.htm)

Support Reference: REDACTED

**Reply by wea-f** [reply]
yeah its blocked for me

**Comment by wea-f** [comment]
just wondering, but why doesn't the School Network Restriction Bypass IP servers don't work for me? Is it because of my version or some admin control?

**Reply by S-PScripts** [reply]
i just looked at the extension that you said was called "xFanatical Safe Doc"... it's actually ridiculous!! it's just TOO SAFE, it actually makes your experience WORSE

**Reply by wea-f** [reply]
yeah ikr

**Reply by AshtonDavies** [reply]
Literally, the Safe Doc extension used to break Google docs, which made students unable to do their work.

**Comment by trentonk23** [comment]
incognito works at the sign in page. power off 2 times than click the blue link

**Reply by wea-f** [reply]
Do you know what the blue link says, because I don't see it at the sign in page.

**Reply by trentonk23** [reply]
It should say visit the Network sign in page,click it  than click the blue link

**Reply by wea-f** [reply]
At the sign in page, the only blue links on the page is "This chromebook is managed by yourdistrict.k12.state.us. [Learn more](https://example.com)" and "[Forgot email?](https://example.com)" I don't see a network sign in page, only a "Network not available" when there is no wifi.

**Reply by trentonk23** [reply]
try these:

 https://nebulaproxy.dev/options/ 

https://proxybrowser.xyz/

https://flipgrid.cf/

https://busy-pink-lamb-hose.cyclic.app/

https://codesandbox.io/s/alloy-proxy-vuobjk

**Reply by wea-f** [reply]
all blocked

**Reply by S-PScripts** [reply]
go here https://github.com/S-PScripts/chromebook-utilities/blob/main/Prxes/Pr*xi*s%20Links

**Reply by wea-f** [reply]
Thanks, the links work (like 3 of them) but I still want to disable the extension.

**Comment by Su4o** [comment]
Well the only thing you can probably do is get your personal computer to school.

If you don't know the password to the school wifi, there are tons of ways to. Here is one that actually worked for me and not like the other ones.

If you are going to powerwash your school chromebook, make sure to use this to get your school wifi password!

1. Go to chrome://net-export
2. In options set the option to Include raw bytes
3. Click Start logging to Disk and save the file
4. Go to chrome://policy in a new tab.
5. Click Reload policies
6. Go back to your other tab.
7. Stop logging.
8. Go to https://luphoria.com/netlog-policy-password-tool
9. Click Choose File
10. Choose the file you saved in Step 3.
11. The credentials column is the wifi password!

Credits: luphoria

**Reply by S-PScripts** [reply]
thats mine! 🤓 🤓 sex

**Reply by Su4o** [reply]
Oh my bad. And thanks. You helped me get my school password :)

**Reply by wea-f** [reply]
chrome://net-export and luphoria.com is blocked

**Comment by S-PScripts** [comment]
closed unless you still need help


https://github.com/S-PScripts/chromebook-utilities/tree/main/Exploits

**Reply by Charlieis1nsan3** [reply]
🤣  I think your a little late 2 YEARS late.

**Reply by jazzymakesstuff** [reply]
"unless you still need help" 💀
